plugins/modules/dnf_versionlock.py

```diff
@@ -179,6 +179,7 @@ def match(entry, pattern):
     m = NEVRA_RE.match(entry)
     if not m:
         return False
+    # indexing a match object with [] is a Python 3.6+ construct
     for name in (
         '%s' % m["name"],
```

Comment:
@felixfontein this construct is Python 3.6+ (accessing re groups in match objects using `[]`).

Reply:
Well, it does not hurt to make it explicit in the docs.